e2etests/test/method_disconnect.js

```diff
@@ -92,7 +93,7 @@ var protocolAndTermination = [
     delayInSeconds: 2
   },
   {
-    testEnabled: true,
+    testEnabled: false,
```

Comment:
>testEnabled: false [](start = 4, length = 18)

please comment why this is disabled, because this is supposed to be temporary because of a bug.  I assume a bunch of the tests are disabled because of protocol limitations, but it would be nice to know when that's not the case.

Reply:
see comment about the connection:closed event

---
In reply to: [145968166](https://github.com/Azure/azure-iot-sdk-node/pull/114#discussion_r145968166) [](ancestors = 145968166)